upload-bundle.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+bundle=output/builtin-actors.car
+curl -k -X POST -F "data=@${bundle};type=application/octet-stream;filename=\"${bundle}\"" -H "Authorization: Bearer $ESTUARY_TOKEN" -H "Content-Type: multipart/form-data" https://shuttle-4.estuary.tech/content/add > output/upload.txt
```

Comment:
Are't you passing the token as env variable `credentials`?

Reply:
ooh, I thought it was named after the env itself. Let me try that.